lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,17 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $user = user_load($data->uid);
+    $this->user = [
+      'drupal_id' => $data->uid,
```

Comment:
do we say `drupal_id` in northstar or `phoenix_id` and `northstar_id` ? 
cc @DFurnes